src/Plugin/RulesAction/DataListItemRemove.php

```diff
@@ -35,7 +35,7 @@ class DataListItemRemove extends RulesActionBase {
   /**
    * {@inheritdoc}
```

Comment:
Doc block should describe the parameters now. See the other converted actions for inspiration.